sycl/test/backend/cuda/primary_context.cpp

```diff
@@ -1,7 +1,8 @@
 // REQUIRES: cuda
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -I%opencl_include_dir -I%cuda_toolkit_include -o %t.out -lcuda -lsycl
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -I%cuda_plugin_dir -I%opencl_include_dir -I%cuda_toolkit_include -o %t.out -lcuda -lsycl
```

Comment:
What is the point of this change and adding XFAIL?

Reply:
The change fixed build step and XFAIL corresponds to execution for now.